transport/UdpEndpoint.cpp

```diff
@@ -142,10 +175,12 @@ void UdpEndpoint::dispatchReceivedPacket(const SocketAddress& srcAddress, memory
             listener = findListener(_iceResponseListeners, transactionId);
             if (listener)
             {
-                // TODO there is still risk of depletion as not all transactions are responded to.
-                // Only when channel is brought down will it clear its transaction listeners
                 _iceResponseListeners.erase(transactionId);
             }
+            else
+            {
+                listener = findListener(_dtlsListeners, srcAddress);
```

Comment:
Why do we ned this? if we didn't get a response in time we should ignore that, right?

Reply:
I changed it so we do not ad listener for stun transaction ids if the remote port is already routed to our transport in dtlsListeners. This saves space in the _iceREsponseListeners and we would have registered the port anyway to get DTLS, RTP.